packages/core/package.json

```diff
@@ -60,7 +60,7 @@
     "koa-route": "^3.2.0",
     "koa-static": "^5.0.0",
     "lodash-es": "^4.17.11",
-    "minimist": "^1.2.0",
+    "minimist": "^1.2.5",
```

Comment:
Shouldn't `package-lock.json` have been changed after updating this package?

Reply:
You're absolutely right.